README.md

```diff
@@ -0,0 +1,174 @@
+# MCS Benchmark Data
+
+MCS Benchmark Data is a set of scripts that can be ued to create and execute Extract, Transform, Load (ETL) pipelines that store information about Machine Commonsense benchmarks into the Commonsense Benchmark Ontology.
+
+For more information about the ontology, refer to:
+[Commonsense Benchmark Ontology](https://tetherless-world.github.io/mcs-ontology/docs/index-en.html)
+*Henrique O. Santos, Minor Gordon, Zhicheng Liang, Gretchen Forbush, Deborah L. McGuinness*
+
+
+## Usage: Adding a Pipeline
+
+Adding a new benchmark to the ontology requires adding a pipeline.
+
+### 1. Add the Appropriate Data
+
+Add a folder with the name of the benchmark (snake_case) to the `data` directory. 

```

Comment:
It seems like most of this could be a script with input()'s. Prompt the user for the benchmark name, ensure it's snake case, create the directories, prompt the user to drop in the files, and so on. At the end you can generate skeleton code.

Reply:
A script is also testable, whereas these instructions are not.

Reply:
You can write the script in Python, it doesn't have to be bash.